tests/qir/test_pytket_qir_14.ll

```diff
@@ -72,28 +72,24 @@ entry:
   call void @set_all_bits_in_reg(i64 %0, i64 %17)
   %18 = lshr i64 %0, 1
   call void @set_all_bits_in_reg(i64 %1, i64 %18)
-  %19 = icmp sgt i64 1, %0
```

Comment:
This file is part of the tests, it is just used to compared the generated qir to what we are expecting